src/NLight/NLight.csproj

```diff
@@ -1,22 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
-    <Version>2.1.1</Version>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Would it make sense to target .NET 6 here as well?

Reply:
I kept the settings to netstandard, so people who are still using classic .net framework are still able to use this library.